tests/pick/pass_ms.js

```diff
@@ -0,0 +1,4 @@
+function pick(o, keys) {
+  for (var k in o) keys.includes(k) || delete o[k];
```

Comment:
that's not good.
Pick should not change the original object.
In general you want to avoid `in` and `delete` in javascript.

We should use something like this:
```js
const pick = (o, keys) => Object.fromEntries(Object.entries(o).filter(([k]) => keys.includes(k))
```

Or 
```js
const pick = (o, keys) => {
  const result = {}
  for (const k of keys) {
    if (o.hasOwnProperty(k)) {
      results[k] = o[k]
    }
  }
  return results
}
```

Or even
```js
const pick = (o, keys) => Object.fromEntries(
  keys
    .filter(k => o.hasOwnProperty(k))
    .map(k => [k, o[k]])
)
```